native/src/components/base/RadioButton.tsx

```diff
@@ -43,7 +19,7 @@ type RadioButtonProps = {
 
 const RadioButton = ({ children, selected, select }: RadioButtonProps): ReactElement => (
   <Container onPress={select} role='radio'>
```

Comment:
❓ Why is this container/pressable still necessary? Also, I think we should just delete this component altogether and use the react-native-paper component directly.

Reply:
See https://callstack.github.io/react-native-paper/docs/components/RadioButton/RadioButtonItem/#mode:
> RadioButton.Item allows you to press the whole row (item) instead of only the RadioButton.